src/actions.ts

```diff
@@ -0,0 +1,43 @@
+// Sample set of actions for authentication. These are the necessary actions in order to make the Auth flow functional in your app.
+interface Form {
+    email: string;
+    username: string;
+    password: string;
+    newsletter: Boolean;
```

Comment:
`boolean`
We don't have a global tslint rule yet, but it would make sens to align all native types